types/attestation.yaml

```diff
@@ -15,6 +15,23 @@ IndexedAttestation:
       $ref: './primitive.yaml#/Signature'
       description: "The BLS signature of the `IndexedAttestation`, created by the validator of the attestation."
 
+SingleAttestation:
+  type: object
+  description: "The [`SingleAttestation`](https://github.com/ethereum/consensus-specs/blob/v1.3.0/specs/electra/beacon-chain.md#singleattestation) object from the CL spec."
```

Comment:
this results in a 404, need to wait for upstream PR to be part of a pre-release, or update it later on